File: app/src/main/java/net/numa08/gochisou/data/model/Team.java
package net.numa08.gochisou.data.model;

import io.realm.RealmObject;

public class Team extends RealmObject {
    private String name;
    private String privacy;
    private String description;
    private String icon;
    private String url;

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getPrivacy() {
        return privacy;
    }

    public void setPrivacy(String privacy) {
        this.privacy = privacy;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getIcon() {
        return icon;
    }

    public void setIcon(String icon) {
        this.icon = icon;
    }

    public String getUrl() {
        return url;
    }

    public void setUrl(String url) {
        this.url = url;
    }
}

File: app/src/main/java/net/numa08/gochisou/data/model/Team.kt
package net.numa08.gochisou.data.model

import org.parceler.Parcel
import org.parceler.ParcelConstructor
import org.parceler.ParcelProperty

@Parcel(Parcel.Serialization.BEAN)
data class Team
@ParcelConstructor
constructor(
        @ParcelProperty("name")
        val name: String,
        @ParcelProperty("privacy")
        val privacy: String,
        @ParcelProperty("description")
        val description: String,
        @ParcelProperty("icon")
        val icon: String,
        @ParcelProperty("url")
        val url: String
)
